__tests__/acceptance/eslint-with-todo-formatter-test.ts

```diff
@@ -1265,4 +1265,1468 @@ describe('eslint with todo formatter', function () {
       }
     });
   }
+
+  describe('with FORMAT_TODO_AS', function () {
+    it('with UPDATE_TODO, outputs empty formatted results from alternate formatter', async () => {
+      project.write({
+        src: {
+          'with-errors-0.js': getStringFixture('with-errors-0.js'),
+          'with-errors-1.js': getStringFixture('with-errors-1.js'),
+        },
+      });
+
+      const result = await runEslintWithFormatter({
+        env: {
+          FORMAT_TODO_AS: '@microsoft/eslint-formatter-sarif',
+          UPDATE_TODO: '1',
+        },
+      });
+
+      expect(result.exitCode).toEqual(0);
+      expect(JSON.parse(result.stdout)).toEqual(
+        expect.objectContaining({
+          runs: [
+            {
+              tool: {
+                driver: expect.objectContaining({
+                  name: 'ESLint',
+                }),
+              },
+              results: [],
+            },
+          ],
+        })
+      );
+    });
+
+    it('with UPDATE_TODO and INCLUDE_TODO, outputs format results from alternate formatter', async () => {
+      project.write({
+        src: {
+          'with-errors-0.js': getStringFixture('with-errors-0.js'),
+          'with-errors-1.js': getStringFixture('with-errors-1.js'),
+        },
+      });
+
+      const result = await runEslintWithFormatter({
+        env: {
+          FORMAT_TODO_AS: '@microsoft/eslint-formatter-sarif',
+          UPDATE_TODO: '1',
+          INCLUDE_TODO: '1',
+        },
+      });
+
+      expect(result.exitCode).toEqual(0);
+      expect(JSON.parse(result.stdout)).toEqual(
+        expect.objectContaining({
```

Comment:
While this is useful in explicitly validating the SARIF output, we can likely just simplify this by using the [`@microsoft/jest-sarif`](https://www.npmjs.com/package/@microsoft/jest-sarif) assertions to ensure that we have valid SARIF.